core/src/main/kotlin/fr/sncf/osrd/api/api_v2/pathfinding/PathfindingBlocksEndpointV2.kt

```diff
@@ -138,79 +147,192 @@ private fun computePaths(
     }
 
     // Handling errors
-    // Check if pathfinding failed due to constraints
+    // Check if pathfinding failed due to incompatible constraints
+    val elapsedSeconds = Duration.between(start, Instant.now()).toSeconds()
```

Comment:
I really don't like this. I see why you're doing it likeso: you want the pathfinding to timeout if constrained pathfinding + unconstrained pathfinding > 2min. But the whole point of the timeout was to interrupt requests that would take way too much time to find a solution. Here, we already know the constrained is <2min, so the unconstrained will take even less time. We risk throwing a timeout exception instead of waiting 10 more seconds to have more details about why the pathfinding failed, which is the goal of this feature.